diff --git a/install.sh b/install.sh
index 312281f..5c5f215 100755
--- a/install.sh
+++ b/install.sh
@@ -32,7 +32,7 @@ else
     $SUDO install -Dm 644 ./systemd-swap.conf $PREFIX/etc/systemd-swap.conf
 fi
 
-$SUDO install -Dm755   ./systemd-swap.sh $PREFIX/usr/lib/systemd/scripts/systemd-swap.sh
+$SUDO install -Dm755   ./systemd-swap.sh "$PREFIX/usr/$(exhost --target)/lib/systemd/scripts/systemd-swap.sh"
 $SUDO mkdir -p $PREFIX/etc/systemd/
 $SUDO ln -svf  /etc/systemd-swap.conf $PREFIX/etc/systemd/swap.conf
-$SUDO install -Dm644 ./systemd-swap.service  $PREFIX/usr/lib/systemd/system/systemd-swap.service
+$SUDO install -Dm644 ./systemd-swap.service  "$PREFIX/usr/$(exhost --target)/lib/systemd/system/systemd-swap.service"
